homeassistant/components/vodafone_station/const.py

```diff
@@ -11,3 +11,5 @@
 DEFAULT_USERNAME = "vodafone"
 
 LINE_TYPES = ["dsl", "fiber", "internet_key"]
+
+UPTIME_DEVIATION = 30
```

Comment:
It is preferred that single-use constants be kept inside the platform

Reply:
Moved